include/odp/api/spec/classification.h

```diff
@@ -856,67 +835,6 @@ uint32_t odp_cls_cos_num_queue(odp_cos_t cos);
  */
 uint32_t odp_cls_cos_queues(odp_cos_t cos, odp_queue_t queue[], uint32_t num);
 
-#if ODP_DEPRECATED_API
-
-/**
- * Assign packet drop policy for specific class-of-service
- *
- * @param cos          CoS handle
- * @param drop_policy  Desired packet drop policy for this class.
- *
- * @retval  0 on success
- * @retval <0 on failure
- *
- * @note Optional.
- */
-int odp_cos_drop_set(odp_cos_t cos, odp_cls_drop_t drop_policy);
-
-/**
-* Get the drop policy configured for a specific class-of-service instance.
-*
-* @param cos           CoS handle
-*
-* @retval Drop policy configured with the given class-of-service
-*/
-odp_cls_drop_t odp_cos_drop(odp_cos_t cos);
-
-#endif
-
-/**
- * Request to override per-port class of service based on Layer-2 priority field if present.
- *
- * @deprecated Use #ODP_PMR_VLAN_PCP_0 instead.
- *
- * @param pktio_in     Ingress port identifier.
- * @param num_qos      Number of QoS levels, typically 8.
- * @param qos_table    Values of the Layer-2 QoS header field.
- * @param cos_table    Class-of-service assigned to each of the allowed
- *                     Layer-2 QOS levels.
- *
- * @retval  0 on success
- * @retval <0 on failure
- */
-int ODP_DEPRECATE(odp_cos_with_l2_priority)(odp_pktio_t pktio_in, uint8_t num_qos,
```

Comment:
`odp_cos_with_l2_priority()` is still mentioned in `doc/users-guide/users-guide-cls.adoc`.

Reply:
V2.